### ISSUE TITLE
Example code single integer ValueError

### ISSUE THREAD

**Issue by FoamyGuy** [adafruit/Adafruit_CircuitPython_FRAM#29]
In the simpletest example there is commented out code that writes multiple bytes to the FRAM device at once:
https://github.com/adafruit/Adafruit_CircuitPython_FRAM/blob/34c3f75f6c86779f4fcd5b0823d0d6fc59abb6b0/examples/fram_spi_simpletest.py#L30-L32

But if you uncomment and attempt to run this code it raises this error:
```
Traceback (most recent call last):
  File "code.py", line 31, in <module>
  File "adafruit_fram.py", line 173, in __setitem__
ValueError: Data must be a single integer for single addresses
```
I think the line writing the data needs to include a slice of the proper length like this:
```py
fram[0:99] = values
```
I tested that locally and it seems to work, though I have very little experience with FRAM so I can't be certain it's correct.

**Comment by tekktrik** [comment]
That's new from the introduction of slices I added, the intention was to be more in line with Issue #26 and make use of slice notation.  You're correct, now that it's implemented, ideally the example should be `fram[0:100] = values` and that should work (which you said, but I think it needs to be `[0:100]` which is accounting for Issue #30.   The default behavior could now be to just take `values[0]` and make that the value of `fram[0]` in this context.

**Comment by tekktrik** [comment]
So I could add that functionality and/or update the example, happy to do whatever makes most sense!

**Comment by FoamyGuy** [comment]
My understanding of #26 is that normal python behavior for:
```
fram[0] = [2,3,4,5]
```
would store the list `[2,3,4,5]` at index `0` in `fram`. If `fram` were a regular list this would be possible since any type of thing can be stored in the indexes.

But because `fram` is not actually a regluar list but instead a list-like API to the hardware storage, essentially the same as  a bytearray it's not possible to lists in the individual indexes. It's only possible to store ints in each of the indexes in frame.
so that same code:
```
fram[0] = [2,3,4,5]
```
Should probably raise some error indicating that you are not allowed to store a list object in that index, only a int.

I think we do need to update the example either way since we don't want it to raise an exception.

I do think it's also good to go ahead update the library as well to raise the error about not being able to store list objects instead of ints like the example shown in #26

**Comment by tekktrik** [comment]
Sounds good!  I'll submit a PR that:

1. Changes the warning to say that only ints can be stored in an index.  I think that was the intention meant to be communicated with this error text, but your phrasing makes much more sense!
2. Updates the example accordingly by removing the problematic line, or stores a list of ints correctly like `fram[0:3] = [1, 2, 3]`

**Comment by FoamyGuy** [comment]
closed by #32
